package.json

```
@@ -56,7 +55,7 @@
     "@types/estree-jsx": "^1.0.0",
     "@types/mdast": "^4.0.0",
     "@types/node": "^22.0.0",
-    "acorn": "^8.0.0",
+    "acorn": "^8.11.0",
```

We don’t need to narrow the semver range. Both ranges resolve to the same version.

```suggestion
    "acorn": "^8.0.0",
```

---

Well, recommended by @ChristianMurphy 

And if the current cloned repo in machine won't upgrade automatically, so I think change the range is reasonable.

---

We keep circling back to this discussion. Both strategies have pros and cons.

Updating the version will force package managers to update the dependency, which on its own is fine. But some package managers insist on locking versions and will duplicate the dependency rather than just update. This leads to excessively big `node_modules` and lock files. It also leads to unnecessary diffs for dependants.

The policy in the unified ecosystem is to use version ranges of the form `^x.0.0`. This is no exception.

---

It's dev dependency, only affect maintainers.

---

I have a bit more mixed feelings on this one.
1. It's a dev dependency
2. It's a minor version that made a major change

---

I'd be fine leaving the full major range though

---

I think that it is expected for developers to do fresh clones and fresh installs, thus getting recent versions from the dependencies, and we don’t always need to update `package.json`s for that